examples/processor/processor/app.py

```diff
@@ -0,0 +1,47 @@
+import asyncio
+import signal
+
+import aiorun
+
+import kstreams
+
+stream_engine = kstreams.create_engine(title="my-stream-engine")
+
+
+@stream_engine.stream(topics=["local--hello-world"], group_id="example-group")
+async def consume(stream):
+    print("Consumer started")
+    try:
+        async for cr in stream:
+            print(f"Event consumed: headers: {cr.headers}, payload: {cr}")
+    finally:
+        # Terminate the program if something fails. (aiorun will cath this signal and properly shutdown this program.)
+        signal.alarm(signal.SIGTERM)
+
+
+async def produce():
+    payload = b'{"message": "Hello world!"}'
+
+    while True:
+        metadata = await stream_engine.send(
+            topic="local--hello-world",
+            value=payload,
+            key="1",
+            headers={},
+        )
+
+        print(f"Message sent: {metadata}")
+        await asyncio.sleep(5)
+
+
+async def start():
+    await stream_engine.start()
+    await produce()
```

Comment:
IMO the producer should not be in this example. We can see the `stream` is running because the `while True` is keeping the software running.

We should instead have in the `README` and example of how to produce using the `scripts/cluster/events/send` script

Reply:
My only concern is that removing the producer would require more work for a user to get up and running. An example consumer isn't very useful without data to consume.

Reply:
That's why the README should include something like:
```
## Sending events

To send events to this consumer run `scripts/cluster/events/send`

and start producing anything you want
```

The challenge for this example is to show how to run a single consumer. More explicitly:

## Use case
The user wants to consume from a topic

----

Your example won't help that use case, because if I just add a consumer following your instructions the program will terminate.

Reply:
I will remove the producer and instruct the user to use the send event script. I will test to make sure the consumer doesn't terminate.